### DISCUSSION TITLE
Recording Includes All Renditions in Output

### DISCUSSION THREAD

**Discussion by maruf-cs** [OvenMediaLabs/OvenMediaEngine#1471] [Help] [answered]
When I initiate recording using the Record Start API, it begins recording through a temporary file. After recording stops, it generates a .ts/.mp4 file. However, the .ts/.mp4 file contains all renditions as a single file. As the screenshot shows 5 video track in a single file. Is it possible to record a specific rendition only, or will each rendition generate a separate .ts/.mp4 file?

![image](https://github.com/AirenSoft/OvenMediaEngine/assets/61920373/87862c5d-01eb-4a87-a1e5-6e9e7f5a5d3a)

**Comment by maruf-cs** [comment]
https://github.com/AirenSoft/OvenMediaEngine/assets/61920373/5620474d-d3c6-4a91-9936-2514f98be81e

Here is a recording output example

**Comment by nums** [comment, marked as answer]
Yes, with the rest api you can manage  what tracks you want to record : https://airensoft.gitbook.io/ovenmediaengine/rest-api/v1/virtualhost/application/recording

## variantNames (optional)
        Array of track names to record. If empty, all tracks will be 
        recorded. This value is Encodes.[Video|Audio|Data].Name in the
        OutputProfile setting.

**Reply by maruf-cs** [reply]
Thank you for pointing that out, I missed that part about the variantNames in the REST API documentation. I appreciate your help!
